### PR TITLE
Update proxy.rst

### PR DESCRIPTION
**Pull request by gebhard73**
Clarification about environment variables when running commands as root.

### CODE REVIEW

File: proxy.rst [hunk 1]
```diff
@@ -29,3 +29,5 @@ If you're going to run something using sudo, remember to use the ``-i`` option t
 ::
 
     sudo -i so-rule-update
+
+It won't work if you do a ``sudo su -`` followed by a ``so-rule-update``.
```
Comment (rwwiv) [inline proxy.rst]: `sudo su -` ignores `/etc/environment`, but `sudo su` does not.
 
I would probably reword this as:
```rst
.. warning::
	Using ``sudo su -`` will ignore ``/etc/environment``, instead use ``sudo su`` if you need to operate as root.
```

**Review by rwwiv — CHANGES_REQUESTED** [review]

**Comment by gebhard73** [comment]
Thanks, this explains it better.

**Review by gebhard73 — COMMENTED** [review]
Thanks, this explains it better.
